eras/alonzo/impl/src/Cardano/Ledger/Alonzo/Tx.hs

```diff
@@ -164,6 +165,8 @@ data AlonzoTx era = AlonzoTx
   }
   deriving (Generic)
 
+instance Inject (AlonzoTx era) (AlonzoTx era)
```

Comment:
Instead of adding a whole bunch of instances like that I think we can just add one `Inject a a`. Could you try my PR: #4350 and see if it'll work for you

Reply:
#4350 is now merged.